pos-v2/contracts/Subscriptions.sol

```diff
@@ -68,6 +75,8 @@ contract Subscriptions is ISubscriptions, Ownable{
         });
         virtualChains[vcid] = vc;
 
+        emit VcOwnerChanged(vcid, owner);
```

Comment:
do we want to emit this here? even if we want to emit "changed" on creation, it should come after the creation event. not before. And i think we should consider removing this altogether